src/Store/CommandStoreConnector.re

```diff
@@ -260,139 +79,15 @@ let start = (getState, contributedCommands) => {
     );
 
   let commands = [
-    ("keyDisplayer.enable", _ => singleActionEffect(EnableKeyDisplayer)),
-    ("keyDisplayer.disable", _ => singleActionEffect(DisableKeyDisplayer)),
```

Comment:
Most of these are now unnecessary as the mapping is done in the command definition itself.

Reply:
Excellent!